### PR TITLE
New youtube-dl cheat sheet

### PR DESCRIPTION
**Pull request by pa1pal**
IA : [youtube-dl](https://duck.co/ia/view/youtube_dl_cheat_sheet)

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pa1pal Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
## [Youtube-Dl Cheat Sheet](https://duck.co/ia/view/youtube_dl_cheat_sheet)

**Description**: youtube-dl is a small command-line program to download videos from YouTube.com and a few more sites.

**Example Query**: [](https://beta.duckduckgo.com/?q=), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: https://github.com/rg3/youtube-dl/blob/master/README.md#readme

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/youtube_dl_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/youtube_dl_cheat_sheet)*

File: share/goodie/cheat_sheets/json/youtubedl.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{
+	"id": "youtubedl_cheat_sheet",
+	"name": "Youtubedl",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/youtubedl.json]: Let's change this to `Youtube-dl`

File: share/goodie/cheat_sheets/json/youtubedl.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+	"description": "Small command-line program to download videos from YouTube.com and other video sites",
+
+	"metadata": {
+		"sourceName": "github.com",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/youtubedl.json]: nit: It looks like you've used hard tabs. Can you switch the indentation over to 4-space soft tabs please?

File: share/goodie/cheat_sheets/json/youtubedl.json [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+		"sourceUrl": "https://github.com/rg3/youtube-dl/blob/master/README.md#readme"
+	},
+
+	"aliases": [
+		"youtubedl", "youtube-dl", "youtube dl"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/youtubedl.json]: please remove `youtubedl` -- that already triggers the Cheat Sheet because it's the filename.

**Comment by moollaza** [comment]
@pa1pal Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by pa1pal** [comment]
Thanks @moollaza . I have some problem with codio I thought subscription was over. But now I am working and interested in finishing this up.

**Comment by jdorweiler** [comment]
ping @moollaza

File: share/goodie/cheat_sheets/json/youtubedl.json [hunk 4]
```diff
@@ -0,0 +1,110 @@
+{
+    "id": "youtubedl_cheat_sheet",
+    "name": "Youtube-dl",
+    "description": "Small command-line program to download videos from YouTube.com and other video sites",
+
+    "metadata": {
+        "sourceName": "github.com",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/youtubedl.json]: Just use `GitHub`.

Comment (pa1pal) [inline share/goodie/cheat_sheets/json/youtubedl.json]: done.

**Comment by GuiltyDolphin** [comment]
@pa1pal Thanks! :dolphin:

**Pull request merged**